pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -1026,6 +1033,64 @@ func PodEnv(i *infinispanv1.Infinispan, systemEnv *[]corev1.EnvVar) []corev1.Env
 	return envVars
 }
 
+func applyExternalArtifactsDownload(ispn *infinispanv1.Infinispan, spec *corev1.PodSpec) (updated bool) {
```

Comment:
WDYT to splitting this logic into a separate `dependencies.go` file in the same package? `infinispan_controller.go` already has a lot of content and with https://github.com/infinispan/infinispan-operator/pull/1007 we will do the same for encryption.

Reply:
Yes, good point. Moving it into the `dependencies.go` file